app/src/main/java/com/alfame/ville/memo/SQLiteDriver.java

```diff
@@ -1,117 +1,318 @@
+
+
+        dbName="notes";
+        idCount=0;
+        tableName="notes";
         idName="ID";
         titleName="Title";
         textName="Text";
         categoryName="Category";
         struckName="Struck";
-        Try getting resources again
-    * */
+
+        System.out.println("Constructor "+ file.getName()+" " +file.getAbsolutePath());
+        createDatabase();
+
+    }
 
     @Override
     public ArrayList loadItemsList() {
-        return null;
+        ArrayList<Note> noteList = new ArrayList<>();
+        String sql;
+
+        //UH OH HARDCODED
+        sql = "SELECT ID, TITLE, TEXT, CATEGORY, STRUCK FROM notes ORDER BY ID";
+        Cursor cl = database.rawQuery(sql,null);
+        cl.moveToFirst();
+        while(!cl.isAfterLast()){
+            //checks if struck
+            Boolean struckB=false;
+            if(!cl.getString(4).equals("FALSE")){
+                struckB=true;
+            }
+
+            //testing
+            System.out.println("loadItemsList :");
+            System.out.println("note ID: "+cl.getInt(0)+ " Title: "+cl.getString(1)+" Text: "+cl.getString(2));
+
+            //id,title,text,category,struck
+            Note newNote = new Note(cl.getInt(0),
+                    cl.getString(1),
+                    cl.getString(2),
+                    cl.getString(3),
+                    struckB);
+            noteList.add(newNote);
+
+            //makes sure idCount stays ahead of last id
+            if(newNote.getID()>idCount){
+                idCount=newNote.getID();
+            }
+            idCount++;
+            cl.moveToNext();
+        }
+        cl.close();
+
+        //prints out all, for testing
+        for(int i=0;i<noteList.size();i++){
+            noteList.get(i).printNote();
+        }
+
+
+        return noteList;
     }
 
     @Override
-    public void addItem(String item) {
+    public void addItem(Note note) {
+
+        /**WHAT'S THIS GREEN SHIT FOR
+         *
+         *
+         * use storageOperations.getidCount() for id when creating a Note so it don't get mixed up?
+         *
+         *
+         * INSERT INTO notes ('id','Title','Text','Category','Struck')
+         * VALUES(int,'String','String','String','String');
+         *
+         *
+         * ' and " etc in strings are going to mess with SQL!
+         */
+
+        try{
+            String sql = "INSERT INTO "+tableName+" " +
+                    "('"+ idName + "', " +
+                    "'"+ titleName+"', " +
+                    "'"+ textName+"', " +
+                    "'"+ categoryName+"', " +
+                    "'"+ struckName+"') " +
+
+                    "VALUES ("
+                    +note.getID()+","+
+                    "'"+note.getTitle()+"'," +
+                    "'"+note.getText()+"'," +
+                    "'"+note.getCategory()+"'," +
+                    "'"+String.valueOf(note.isStruck()).toUpperCase()+"'"+
```

Comment:
Boolean values in sqlite are usually stored in integers 0 and 1. No need to fix if you don't want.